libtcmu.c

```diff
@@ -544,34 +544,37 @@ static void remove_device(struct tcmulib_context *ctx,
 static int is_uio(const struct dirent *dirent)
 {
 	int fd;
-	char tmp_path[64];
-	char buf[256];
-	ssize_t ret;
+	char *tmp_path;
+	char buf[256] = {'\0'};
```

Comment:
No this is on tmp_path, may be the buf fix is irrelevant to topic here :-)
I found them while performing rpmbuild.

```
/root/rpmbuild/BUILD/tcmu-runner-1.2.0/libtcmu.c: In function 'open_devices':    
/root/rpmbuild/BUILD/tcmu-runner-1.2.0/libtcmu.c:593:56: warning: '%s' directive output may be truncated writing up to 255 bytes into a region of size 49 [-Wformat-truncation=]
   snprintf(tmp_path, sizeof(tmp_path), "/sys/class/uio/%s/name",                
                                                        ^~                       
In file included from /usr/include/stdio.h:939:0,                                
                 from /usr/include/libnl3/netlink/netlink.h:15,                  
                 from /usr/include/libnl3/netlink/genl/genl.h:15,                
                 from /root/rpmbuild/BUILD/tcmu-runner-1.2.0/libtcmu.c:33:       
/usr/include/bits/stdio2.h:64:10: note: '__builtin___snprintf_chk' output between 21 and 276 bytes into a destination of size 64
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,           
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~           
        __bos (__s), __fmt, __va_arg_pack ());                                   
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                            
```